pkg/protocols/headless/request.go

```diff
@@ -183,7 +183,7 @@ func (request *Request) executeRequestWithPayloads(input *contextargs.Context, p
 		responseBody, _ = html.HTML()
 	}
 
-	outputEvent := request.responseToDSLMap(responseBody, out["header"], out["status_code"], reqBuilder.String(), input.MetaInput.Input, navigatedURL, page.DumpHistory())
+	outputEvent := request.responseToDSLMap(responseBody, out["header"].(string), out["status_code"].(string), reqBuilder.String(), input.MetaInput.Input, navigatedURL, page.DumpHistory())
```

Comment:
I'm going to leave a note here: also, this `status_code` output should be an integer type _(break change)_ - but first I'm going to check out and see whether this has been used _(as DSL expr)_ before.